src/render_iframe.js

```diff
@@ -267,12 +275,40 @@ EPUBJS.Render.Iframe.prototype.isElementVisible = function(el){
 	return false;
 };
 
+EPUBJS.Render.Iframe.prototype.scroll = function(boolX, boolY, type){
```

Comment:
Added as "flow" in view_managers: https://github.com/futurepress/epub.js/blob/view_managers/src/rendition.js#L313